packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -361,6 +363,16 @@ class SelectableText extends StatefulWidget {
   /// Otherwise [ColorScheme.primary] of [ThemeData.colorScheme] is used.
   final Color? cursorColor;
 
+  /// The color to use when painting the selection.
```

Comment:
This doc could use a bit of rewording in my opinion. A suggestion, taking some inspiration from the TextSelectionThemeData docs:

1) Per the style guide we start with one short phrase that explains the parameter (this line is taken from the TextSelectionThemeData docs)
2) The phrase with the "defaults to" can be a bit shorter I think?
3) What color the default actually is (for example semi-transparent grey) should be omitted, since that is an impementation detail. A user could for example set up a different default in their ThemeData

```
The background color of selected text.

If this is null, the background color of selected text defaults to [DefaultSelectionStyle.defaultColor].

This is ignored if there is no ambient [SelectionContainer].
```

Reply:
Hi, I think this documentation probably came from `Text.selectionColor`. The paragraph that mentions `SelectionContainer` should not be included since it is not used by `SelectableText`.

I think the documentation from `EditableText.selectionColor` is more appropriate considering `SelectableText` implementation.

```
The color to use when painting the selection.

If this property is null, this widget gets the selection color from the inherited [DefaultSelectionStyle] (if any); if none, the selection color is derived from the [CupertinoThemeData.primaryColor] on Apple platforms and [ThemeData.colorScheme.primary] on other platforms with 40% opacity.
```

Reply:
Thank you for your suggestions! I have updated it 👍🏽